proposals/zips/zip001.sol

```diff
@@ -99,88 +56,48 @@ contract zip001 is Proposal, TimelockProposal {
         _core.setGlobalLock(addresses.getAddress("GLOBAL_REENTRANCY_LOCK"));
 
         // Set LOCKER role for all NFT minting contracts
-        _core.grantRole(Roles.LOCKER, addresses.getAddress("ERC1155_MAX_SUPPLY_MINTABLE_CONSUMABLES"));
-        _core.grantRole(Roles.LOCKER, addresses.getAddress("ERC1155_MAX_SUPPLY_MINTABLE_PLACEABLES"));
         _core.grantRole(Roles.LOCKER, addresses.getAddress("ERC1155_MAX_SUPPLY_MINTABLE_WEARABLES"));
-        _core.grantRole(Roles.LOCKER, addresses.getAddress("ERC1155_AUTO_GRAPH_MINTER"));
 
         /// MINTER role for all NFT minting contracts
-        _core.grantRole(Roles.MINTER, addresses.getAddress("ERC1155_MAX_SUPPLY_MINTABLE_CONSUMABLES"));
-        _core.grantRole(Roles.MINTER, addresses.getAddress("ERC1155_MAX_SUPPLY_MINTABLE_PLACEABLES"));
         _core.grantRole(Roles.MINTER, addresses.getAddress("ERC1155_MAX_SUPPLY_MINTABLE_WEARABLES"));
-        _core.grantRole(Roles.MINTER, addresses.getAddress("ERC1155_AUTO_GRAPH_MINTER"));
     }
 
-    function _afterDeployOnChain(Addresses, address deployer) internal override {}
+    function _afterDeployOnChain(Addresses, address deployer) internal override {
+        // Revoke ADMIN role from deployer
+        _core.revokeRole(Roles.ADMIN, deployer);
+    }
 
     function _validate(Addresses addresses, address) internal override {
         /// Check Roles
         assertEq(_core.hasRole(Roles.ADMIN, addresses.getAddress("ADMIN_MULTISIG")), true);
 
         /// Verfiy all contracts are pointing to the correct core address
-        {
-            assertEq(
-                address(GlobalReentrancyLock(addresses.getAddress("GLOBAL_REENTRANCY_LOCK")).core()),
-                address(_core)
-            );
-
-            assertEq(
-                address(
-                    ERC1155MaxSupplyMintable(addresses.getAddress("ERC1155_MAX_SUPPLY_MINTABLE_CONSUMABLES")).core()
-                ),
-                address(_core)
-            );
-            assertEq(
-                address(
-                    ERC1155MaxSupplyMintable(addresses.getAddress("ERC1155_MAX_SUPPLY_MINTABLE_PLACEABLES")).core()
-                ),
-                address(_core)
-            );
-            assertEq(
-                address(ERC1155MaxSupplyMintable(addresses.getAddress("ERC1155_MAX_SUPPLY_MINTABLE_WEARABLES")).core()),
-                address(_core)
-            );
-            assertEq(
-                address(ERC1155AutoGraphMinter(addresses.getAddress("ERC1155_AUTO_GRAPH_MINTER")).core()),
-                address(_core)
-            );
-            assertEq(address(CoreRef(addresses.getAddress("GAME_CONSUMABLE")).core()), address(_core));
-        }
+        assertEq(address(GlobalReentrancyLock(addresses.getAddress("GLOBAL_REENTRANCY_LOCK")).core()), address(_core));
```

Comment:
We might want to perform additional checks to ensure that `CoreRef` is setup correctly, e.g.:

```
assertEq(address(CoreRef(addresses.getAddress("GLOBAL_REENTRANCY_LOCK")).core()), address(_core));
...
assertEq(address(CoreRef(addresses.getAddress("ERC1155_MAX_SUPPLY_MINTABLE_WEARABLES")).core()), address(_core));
```